dataplicity/m2m/dispatcher.py

```diff
@@ -98,4 +98,4 @@ def dispatch_packet(self, packet):
 
     def on_missing_handler(self, packet):
         """Called when no handler is available to handle `packet`"""
-        self.log.error('missing handler for %r', packet)
+        self.log.debug('missing handler for %r', packet)
```

Comment:
By default we want to silently ignore unknown packets. So we can add new packets in the future without spamming logs.